src/ui/component/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,24 @@
+import type { FC } from 'react'
+import classNames from 'classnames'
+import './skeleton.scss'
+
+type SkeletonProps = {
+  width?: number
+  height?: number
+  fontSize?: number
+  variant?: 'text' | 'rectangular' | 'rounded' | 'circular'
+}
+
+export const Skeleton: FC<SkeletonProps> = ({ width, height, variant = 'text', fontSize }) => {
+  const classSkeleton = classNames('okp4-dataverse-portal-skeleton', variant)
+  return (
+    <div
+      className={classSkeleton}
+      style={{
+        width: width ? `${width}px` : '100%',
+        height: `${height}px`,
+        fontSize: `${fontSize}px`
```

Comment:
if no `fontSize` -> `undefinedpx`.

Reply:
React catch it and remove height and fontSize if the value is undefined.
So in CSS there is no height and no font-size
I can cover it with a 100%